ui/ducks/send/send.js

```diff
@@ -231,25 +237,30 @@ async function estimateGasLimitForSend({
     }
   }
 
-  // If we do not yet have a gasLimit, we must call into our background
-  // process to get an estimate for gasLimit based on known parameters.
-
-  paramsForGasEstimate.gas = addHexPrefix(
-    multiplyCurrencies(blockGasLimit, 0.95, {
-      multiplicandBase: 16,
-      multiplierBase: 10,
-      roundDown: '0',
-      toNumericBase: 'hex',
-    }),
-  );
+  if (!isSimpleSendOnNonStandardNetwork) {
+    // If we do not yet have a gasLimit, we must call into our background
+    // process to get an estimate for gasLimit based on known parameters.
+
+    paramsForGasEstimate.gas = addHexPrefix(
+      multiplyCurrencies(blockGasLimit, 0.95, {
+        multiplicandBase: 16,
+        multiplierBase: 10,
+        roundDown: '0',
+        toNumericBase: 'hex',
+      }),
+    );
+  }
+
+  const bufferMultiplier = isSimpleSendOnNonStandardNetwork ? 1 : 1.5;
```

Comment:
Would it be possible to add a comment explaining why we do this?

Reply:
@adonesky1 added in the latest commit